yt/frontends/amrvac/tests/test_outputs.py

```diff
@@ -21,118 +26,113 @@
 rmi_cartesian_dust_2D = "amrvac/Richtmyer_Meshkov_dust_2D/RM2D_dust_Kwok0000.dat"
 
 
-def _get_fields_to_check(ds):
-    fields = ["density", "velocity_magnitude"]
+ds_list = [
+    blastwave_spherical_2D,
+    khi_cartesian_2D,
+    khi_cartesian_3D,
+    jet_cylindrical_25D,
+    riemann_cartesian_175D,
+    blastwave_cartesian_3D,
+    blastwave_polar_2D,
+    blastwave_cylindrical_3D,
+    rmi_cartesian_dust_2D,
+]
+a_list = [0, 1, 2]
+w_list = [None, "density"]
+dso_list = [None, ("sphere", ("max", (0.1, "unitary")))]
```

Comment:
I don't have a clue what those lists represent or are for (even after reading further in). Rather than having their type `list` as part of their names, I would like to have descriptive names here.